core/pom.xml

```diff
@@ -803,6 +805,10 @@ THE SOFTWARE.
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
This has made

    mvn -f jenkinsci/jenkins -DskipTests clean install

far slower (~55s longer on my laptop), making core development less productive. Can we please find a way to suppress this plugin execution during developer builds?

(We need to do the same in `plugin-pom`.)